README.md

```diff
@@ -34,24 +34,38 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. If you were to describe semantic HTML to the next cohort of students, what would you say?
 
+Semantic HTML is just HTML with meaning.
```

Comment:
✔️ 
Yes, the HTML tags convey some meaning about what content is there.  Good job!